web/src/components/core/chip/MqChip.tsx

```diff
@@ -21,41 +20,33 @@ interface MqChipProps {
   selectable?: boolean
 }
 
-const styles = (theme: Theme) =>
-  createStyles({
-    root: {
-      display: 'inline-block',
-      borderRadius: theme.spacing(2),
-      padding: '2px 8px',
-      cursor: 'pointer',
-      userSelect: 'none'
-    },
-    icon: {
-      display: 'inline'
-    },
-    selected: {
-      boxShadow: `0 0 2px 3px ${theme.palette.common.white}`
-    }
-  })
-
 /**
  * This is a simple button that can be either selected or unselected. Is is configurable with icons and/or text
  */
-const MqChip: React.FC<MqChipProps & WithStyles<typeof styles>> = ({
+const MqChip: React.FC<MqChipProps> = ({
   selected,
   onSelect,
   icon,
   text,
   value,
-  classes,
   foregroundColor,
   backgroundColor,
   selectable
 }) => {
+
+  const theme = createTheme(useTheme())
+
   return (
     <Box
       id={`chip-${value}`}
-      className={classNames(classes.root, selected && classes.selected)}
+      sx={Object.assign({
```

Comment:
Do we need to do Object assign here instead of just directly applying the object?

Reply:
Fix